experimental/traffic-portal/src/app/core/profiles/profile-table/profile-table.component.ts

```diff
@@ -115,6 +115,11 @@ export class ProfileTableComponent implements OnInit {
 				}
 			}
 		);
+		const ID = this.route.snapshot.paramMap.get("id");
+		if (ID === null) {
+			return;
+		}
+		this.profiles = this.api.getProfilesByParam(+ID);
```

Comment:
Won't this leave us unable to filter by ID, as it'll be interpreted as a parameter ID? I'd suggest using a query string parameter like "hasParameter={{number}}".